src/app/GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -696,6 +696,9 @@ private void GitcommandLogToolStripMenuItemClick(object sender, EventArgs e)
             FormGitCommandLog.ShowOrActivate(this);
         }
 
+        bool IRevisionGridFileUpdate.SelectFileInRevision(ObjectId objectId, RelativePath ignoredFilename)
+            => RevisionGrid.SetSelectedRevision(objectId);
```

Comment:
I suggest to rename objectId -> commitId (as revisionId is not much used), that describes the functionality better.
At least in the added calls, maybe all calls